Libraries/DirectMLX.h

```diff
@@ -3591,7 +3591,7 @@ namespace dml
         }
         
         TensorDesc outputROIGradientTensor = computeOutputROIGradient ? TensorDesc(roiTensor.dataType, roiTensor.sizes, builder->GetTensorPolicy()) : TensorDesc();
-        assert(!computeOutputROIGradient || computeOutputROIGradient.sizes == roiTensor.sizes);
+        assert(!computeOutputROIGradient || outputROIGradientTensor.sizes == roiTensor.sizes);
```

Comment:
Hmm, how did this even build before? 😂

Reply:
MSVC is pretty chill when compiling release builds and doesn't even look at asserts. I bet this was WSL. :)

Reply:
Yep WSL caught that